io.github.aminbhst.Brisk.yml

```diff
@@ -0,0 +1,61 @@
+id: io.github.aminbhst.Brisk
```

Comment:
https://github.com/Brisk-Apps/brisk is the repo https://docs.flathub.org/docs/for-app-authors/requirements#code-hosting